packages/terra-demographics-banner/lib/DemographicsBanner.js

```diff
@@ -0,0 +1,269 @@
+      var _props = this.props,
+          age = _props.age,
+          additionalDetails = _props.additionalDetails,
+          applicationRows = _props.applicationRows,
+          dateOfBirth = _props.dateOfBirth,
+          deceasedDate = _props.deceasedDate,
+          gender = _props.gender,
+          gestationalAge = _props.gestationalAge,
+          identifiers = _props.identifiers,
+          personName = _props.personName,
+          photo = _props.photo,
+          postMenstrualAge = _props.postMenstrualAge,
+          preferredFirstName = _props.preferredFirstName,
+          customProps = _objectWithoutProperties(_props, ['age', 'additionalDetails', 'applicationRows', 'dateOfBirth', 'deceasedDate', 'gender', 'gestationalAge', 'identifiers', 'personName', 'photo', 'postMenstrualAge', 'preferredFirstName']);
+
+      var mainClasses = (0, _classnames2.default)('terra-DemographicsBanner', { 'terra-DemographicsBanner--deceased': deceasedDate }, customProps.className);
+
+      delete customProps.className;
+
+      return _react2.default.createElement(
+        'section',
+        _extends({ className: mainClasses }, customProps),
+        _react2.default.createElement(
+          'div',
+          { className: 'terra-DemographicsBanner--profile-photo' },
+          this.props.photo
+        ),
+        _react2.default.createElement(
+          'div',
+          { className: 'terra-DemographicsBanner--content' },
+          _react2.default.createElement(
+            'div',
+            { className: 'terra-DemographicsBanner--row' },
+            _react2.default.createElement(
+              'h1',
+              { className: 'terra-DemographicsBanner--person-name' },
+              personName,
+              preferredFirstName && _react2.default.createElement(
+                'span',
+                { className: 'terra-DemographicsBanner--preferred-first-name' },
+                preferredFirstName
+              )
+            ),
+            _react2.default.createElement(
+              'div',
+              { className: 'terra-DemographicsBanner--application-content' },
+              additionalDetails
+            )
+          ),
+          _react2.default.createElement(
+            'div',
+            { className: 'terra-DemographicsBanner--row' },
+            _react2.default.createElement(
+              'div',
+              { className: 'terra-DemographicsBanner--person-details' },
+              this.personDetails()
+            ),
+            _react2.default.createElement(
+              'div',
+              { className: 'terra-DemographicsBanner--identifiers' },
+              this.applicationIdentifiers()
+            )
+          ),
+          this.additionalApplicationRows()
+        )
+      );
+    }
+  }]);
+
+  return DemographicsBanner;
+}(_react2.default.Component);
+
+DemographicsBanner.propTypes = propTypes;
+DemographicsBanner.defaultProps = defaultProps;
+
+exports.default = DemographicsBanner;
```

Comment:
It's node package folder structure. You can see more of it here. https://docs.npmjs.com/files/package.json